fabfile.py

```diff
@@ -104,6 +104,14 @@ def sh(c, service="web"):
     subprocess.run(["docker-compose", "exec", service, "bash"])
 
 
+@task
```

Comment:
what was the reason for adding this one? (I can see it is in kit, so fine for it to be added - just curious)

Reply:
I mainly use it for debugging BE stuff.

For example, if something in Wagtail triggered an error, I would `fab sh-root`, install vim (`apt-get update && apt-get install vim`), then using vim, edit the Wagtail file throwing the error to add `import pdb; pdb.set_trace()` which adds a breakpoint at the line I add it to.

Then when I next `djrun` or `honcho start` and do whatever action is triggering the error again, the breakpoint will take effect, allowing me to look at the values of each variable before the error gets thrown.

e.g.
```
fab sh-root
apt-get update && apt-get install vim
vim /venv/lib/python3.11/site-packages/wagtail/wagtail_hooks.py
```

Reply:
ah thank you!